cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mysql/MySQLBackendImpl.java

```diff
@@ -73,13 +73,8 @@ protected MySQLDriver getDriver() {
         return driver;
     } // getDriver
     
-    /**
-     * Creates a database, given its name, if not exists.
-     * @param dbName
-     * @throws Exception
-     */
     @Override
```

Comment:
Because this method overrides another one declared in `MySQLBackend` interface, where the javadoc section already exists.